core/contract/src/main/kotlin/org/http4k/contract/openapi/v3/OpenApi3.kt

```diff
@@ -315,18 +316,28 @@ class OpenApi3<NODE : Any>(
     }
 
     private fun HttpMessageMeta<HttpMessage>.toSchemaContent(): BodyContent {
-        fun exampleSchemaIsValid(schema: JsonSchema<NODE>) =
-            when (example) {
-                is Array<*>, is Iterable<*> -> !json.fields(schema.node).toMap().containsKey("\$ref")
-                else -> apiRenderer.toSchema(object {}, refModelNamePrefix = schemaPrefix) != schema
-            }
-
-        val jsonSchema = example
-            ?.let { apiRenderer.toSchema(it, definitionId, schemaPrefix) }
-            ?.takeIf(::exampleSchemaIsValid)
-            ?: message.bodyString().toSchema(definitionId)
+        fun exampleSchemaIsValid(schema: JsonSchema<NODE>) = when (example) {
+            is Array<*>, is Iterable<*> -> !json.fields(schema.node).toMap().containsKey("\$ref")
+            else -> apiRenderer.toSchema(object {}, refModelNamePrefix = schemaPrefix) != schema
+        }
 
-        return SchemaContent(jsonSchema, message.bodyString().safeParse())
+        return when (example) {
+            is InputStream -> NoSchema(
+                schema = json {
```

Comment:
Is there a type we can use instead of manually creating the JSON here?

Reply:
Hmm, I used the same approach as in other places in this file:
e.g. 
`val noSchema = consumes.map { it.value to NoSchema(json { obj("type" to string(StringParam.value)) }) }`

Reply:
There is space to refactor this code for sure, but probably we prefer small MRs ;)

Reply:
ok - I was just checking and not wanting to go and look!